src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+
+/*
+ * This is a copy of the form system RadioWidget, but with custom
+ * code to disable certain options. This isn't currently supported by the
+ * form system.
+ */
+export default function FacilitiesRadioWidget({
+  options,
+  value,
+  onChange,
+  id,
+  formContext,
+}) {
```

Comment:
This custom radio widget is needed since our enumOptions are dynamic and require some additional JSX to display the city/state on the 2nd line.  We also use it to display the expand link.